src/librustdoc/clean/auto_trait.rs

```diff
@@ -568,10 +568,10 @@ impl<'a, 'tcx> AutoTraitFinder<'a, 'tcx> {
                                 // Convert something like '<T as Iterator::Item> = u8'
                                 // to 'T: Iterator<Item=u8>'
                                 GenericArgs::AngleBracketed { ref mut bindings, .. } => {
-                                    bindings.push(TypeBinding {
+                                    bindings.extend([TypeBinding {
```

Comment:
Why this change?

Reply:
ThinVec doesn't provide a push method. Should I add it?

Reply:
Yes, please.